src/voice-broadcast/models/VoiceBroadcastPlayback.ts

```diff
@@ -131,8 +131,8 @@ export class VoiceBroadcastPlayback
     private addChunkEvent = async (event: MatrixEvent): Promise<boolean> => {
         const eventId = event.getId();
 
-        if (!eventId
-            || eventId.startsWith("~!") // don't add local events
+        if (
+            !eventId
```

Comment:
Why/when would the event not have an event ID?

Reply:
Why: Technically it's allowed https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/models/event.ts#L395

When: No idea :D

Reply:
In this particular place everything else now works with transaction or event Ids.
The check here is now unnecessary.

Reply:
Room events always have an event ID, its account data events and to-device events which don't, hence the model being looser. Could've used `event?.startsWith("~!")`